jujugui/static/gui/src/app/views/topology/relation.js

```diff
@@ -677,8 +677,6 @@ YUI.add('juju-topology-relation', function(Y) {
         // TODO clean up, abstract
         self.cancelRelationBuild();
       }
-      // Signify that the relation drawing has ended.
-      topo.fire('addRelationEnd');
```

Comment:
That's correct! (Sorry, I did leave a note in the PR description, but it's not really obvious with all the comments/label updates etc.)